packages/platforms/accton/x86-64/as9726_32d/modules/builds/Makefile

```diff
@@ -0,0 +1,6 @@
+KERNELS := onl-kernel-4.19-lts-x86-64-all:amd64
```

Comment:
Why not the latest Linux kernel version?

Reply:
our customers need the 4.19 kernel.

Reply:
Please mention that in the commit message. (I am very uneasy about the fact, that users rely on such old versions, even if they are LTS series. For ONL the latest Linux kernel version should always be usable.)